src/NLog/Internal/Fakeables/IFileSystem.cs

```diff
@@ -33,14 +33,18 @@
 
 namespace NLog.Internal.Fakeables
 {
+    using System.Xml;
+
     /// <summary>
-    /// Abstract calls to File
+    /// Abstract calls to FileSystem
     /// </summary>
-    internal interface IFile
+    internal interface IFileSystem
```

Comment:
FYI, this was not the direction I would take. I think it's better to have 10 small interfaces then 3 medium. 

Also, with the introduction of NSubstitute, really small interfaces are nice. 

But as it's internal I think this is more than good enough 👍